src/stores/SelectedStore.ts

```diff
@@ -77,6 +77,14 @@ export class SelectedStore {
 		localStorageWorker.setPersistedPinnedMessages(this.pinnedMessages);
 	};
 
+	@action
+	public removePinnedMessageFromLocalStorage = (removedMessage: EventMessage) => {
```

Comment:
maybe we'll just have a reaction on pinnedMessagesChanges which updaes pinned messages to localStorage?